runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -1294,8 +1244,6 @@ public synchronized CorfuRuntime connect() {
             }
         }
 
-        checkVersion();
```

Comment:
How will the runtime become aware of the version of the nodes in the cluster? Will it need to?

Reply:
Currently the server version is contained in the response header, and for now only the ClientHandshakeHandler will be aware of it.